Deprecate alternative names on secrets (#3406)
woodpecker-ci/woodpecker: docs/docs/20-usage/40-secrets.md

+    secrets: [ docker_username, docker_password ]
```

### Use secrets in settings and environment

> Alternatively, you can get a setting or environment from secrets using the `from_secret` syntax. This also allows using a different name for the environment variable.

Reviewer: ```suggestion
You can set an setting or environment value from secrets using the `from_secret` syntax.
```
